### ISSUE TITLE
Bug: action will run on previous selected item when run action from right click menu.

### ISSUE THREAD

**Issue by mics8128** [a-tak/auto-bucket#102]
## env
* Arch linux + KDE plasma
* Thunderbird 68.10.0

## reproduce
1. found 2 emails not judged. (named A & B)
2. left clicked on A, make sure it selected.
3. right clicked on B, and select AutoBucket -> Judge this email.

## actual result
* A will be Judged. and no action on B.

## expected result
* B will be Judged.

**Comment by a-tak** [comment]
Certainly, in the current implementation, the processing is performed for the mail selected when "mail judgment" is executed, so that is the behavior.

I will search the API specifications of Thunderbird if the right-clicked mail can be included in the judgment target.
